client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -70,35 +69,41 @@ export default React.createClass( {
 				showStore: true
 			} );
 
-			if ( this._pressableStore ) {
-				this._pressableStore.focus();
+			if ( this.pressableStore ) {
+				this.pressableStore.focus();
```

Comment:
are we simply checking if the object exists? **lodash** provides a safe-defaulting function to do this: `invoke()`

```js
import { invoke } from 'lodash';

invoke( this, 'pressableStore.focus' );
```

that ^^^ will run `this.pressableStore.focus()` if and only if `this.pressableStore.focus` exists as a function